tests/test_app.py

```diff
@@ -0,0 +1,11 @@
+from webob import Request, Response
+from firefly.app import FireflyFunction
+
+def square(a):
+    return a**2
+
+def test_firefly_function_response():
+    func = FireflyFunction(square)
+    request = Request.blank("/", POST='{"a": 1}')
+    response = func(request)
+    assert response.status == '200 OK'
```

Comment:
Usually tests for classes are written in a class.

```
class TestFireflyFunction:
     def test_call(self):
        ....
```

Also assert the response text.